src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -52,7 +59,13 @@ public void SarifLogger_RedactedCommandLine()
                 string pathToExe = Path.GetDirectoryName(Assembly.GetCallingAssembly().Location);
                 string commandLine = Environment.CommandLine;
 
-                if (pathToExe.IndexOf(@"\Extensions", StringComparison.OrdinalIgnoreCase) != -1)
+                if (commandLine.Contains("testhost.dll") || commandLine.Contains("\\xunit.console"))
+                {
+                    var index = commandLine.LastIndexOf("\\");
```

Comment:
>var [](start = 20, length = 3)

don't use var to hold assignment from method calls, prefer the explicit type (so that readers know what it is). Fix this issue everywhere. :)